ckan/logic/action/delete.py

```diff
@@ -37,10 +37,22 @@ def user_delete(context, data_dict):
     user_id = _get_or_bust(data_dict, 'id')
     user = model.User.get(user_id)
 
+    # New revision, needed by the member table
+    rev = model.repo.new_revision()
+    rev.author = context['user']
+    rev.message = _(u' Delete User: {0}').format(user.name)
+
     if user is None:
         raise NotFound('User "{id}" was not found.'.format(id=user_id))
 
     user.delete()
+
+    user_memberships = model.Session.query(model.Member).filter(
+        model.Member.table_id == user_id).all()
+
+    for membership in user_memberships:
+        membership.delete()
```

Comment:
Is this `delete` doing something special in one of the three superclasses? Otherwise we could just use:

```
user_memberships = model.Session.query(
    model.Member
).filter(
        model.Member.table_id == user_id
).delete()
```